pyoptsparse/postprocessing/OptView_baseclass.py

```diff
@@ -192,15 +206,28 @@ def DetermineMajorIterations(self, db, OpenMDAO):
                 # actual major iteration. In particular, one has funcs
                 # and the next has funcsSens, but they're both part of the
                 # same major iteration.
+                # For IPOPT, it saves info for four calls for every
+                # actual major iteration: objective, constraints,
+                # and sensitivities of each.
+
                 if "funcs" in db[key].keys():
+                    # check if this entry is duplicated info. Only relevant for IPOPT.
+                    # Note: old hist files don't have "iter"
+                    if "iter" in db[key].keys() and db[key]["iter"] == previousIterCounter:
+                        # duplicated info, discard
+                        self.iter_type[i] = 0
```

Comment:
I think this simply means that this specific iter is ignored by Optview ([see this block(?)](https://github.com/mdolab/pyoptsparse/blob/b9550644bf898112422593cdb742097f139a5df7/pyoptsparse/postprocessing/OptView_baseclass.py#L225-L239)), so yeah, maybe "discarding" is ambiguous because the db is not touched, maybe "ignored" would be better?

Reply:
Right "ignore" is more accurate here